app/hotels/src/types/datastructures.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+
+import type { LatLng } from 'expo';
+
+export type HotelMarker = {|
+  id: string,
+  coordinate: LatLng,
+  data: {|
+    price: number,
+  |},
+|};
+
+export type CurrencyInfo = {|
+  currency: string,
+  position: 'before' | 'after',
```

Comment:
Now that I think about it, the design in Zeplin shows price with the ISO currency code. That would be always "after".

Airbnb uses local currency symbol placed "before", because they are Americans, but "Kč1,000" is IMHO just wrong.

If we decide to complicate it more in the future, note that on the Frontend we sometimes render something before as well as after the amount, typically for different dollars ("$100 CAD" vs "$100 AUD").

Reply:
My main concern is: do we know what is before and what after or do we have to figure it on our own? Also very good point with the dollars...

Reply:
Ohh, yes, I have difficulty how to grasp this formatting correctly, as I don't know how data will look, I will probably just remove all this formatting stuff for now and solve it later when it will be more clear.